Update wireguard-client.rst to clarify use of WireGuard net (#304)
opnsense/docs: source/manual/how-tos/wireguard-client.rst

=========================== ================ =====================================================================
 **Interface**               WireGuard        *The interface this rule applies to*
 **Source**                  WireGuard net    *Source subnet - but see the warning below*
 **Destination**             any              *Traffic destination*
 **Description**             WG WAN to LAN    *Optional - provide a description*
=========================== ================ =====================================================================

.. Warning::

>     Using **WireGuard net** as the Source will not work as expected if multiple local WireGuard peers or multiple WireGuard endpoints are configured. In those cases, define an Alias that includes the IP addresses (for example, 10.10.10.2 and 10.10.10.3) or subnet (for example, 10.10.10.0/24) of the VPN clients that are intended to have LAN access. Then include that Alias as the Source in the firewall rule.

Reviewer: ```suggestion
    Using **WireGuard net** as source will not work as expected if multiple local WireGuard instances or WireGuard endpoints are configured. In those cases, define an alias including the IP addresses (e.g. 10.10.10.2 and 10.10.10.3) or subnet (e.g. 10.10.10.0/24) of the VPN clients that are intended to have access.
```

Reviewer: I'm not a native speaker but would write it like this.

Author: These largely look like stylistic changes, which I am not going to get upset about (even though I am a native speaker 😀). Deleting the last sentence I included does however mean the instructions only explicitly tell the user to define an Alias, but not to use it in the rule. Implicit (and obvious), sure, but some people seem to need things spelled out.
